Algorithm-Spinal-Tap-Case.md

```diff
@@ -1,30 +1,43 @@
 # Algorithm Spinal Tap Case
 
-### Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
 
-- Convert the given string to an all lowercase sentence joined by dashes.
+### :checkered_flag: Problem Explanation:
 
-## Hint: 1
+Convert the given string to a lowercase sentence with words joined by dashes.
 
-- Create a regex to for all white spaces and underscores.
+#### Relevant Links
 
-## Hint: 2
+- [String global object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+- [JS Regex Resources](JS-Regex-Resources)
+- [JS String Prototype Replace](JS-String-Prototype-Replace)
+- [JS String Prototype ToLowerCase](JS-String-Prototype-ToLowerCase)
 
-- You will also have to make everything lowercase.
+## :speech_balloon: Hint: 1
 
-## Hint: 3
+Create a regex for all white spaces and underscores.
```

Comment:
Ah, thought about it and figured out since it was used everywhere, didn't need to change it. Done 👍